maintenance_tasks.gemspec

```
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency("actionpack", ">= 6.0")
   spec.add_dependency("activejob", ">= 6.0")
   spec.add_dependency("activerecord", ">= 6.0")
+  spec.add_dependency("csv", ">= 3.3")
```

Is there anything specific about this version that we need to require it?

---

It is just the latest version as of today - https://rubygems.org/gems/csv/versions/3.3.0?locale=en. I'm not sure if this specific version is what will no longer exist in 3.4.0, but assuming so.

---

Let's not add any version requirement then.
```suggestion
  spec.add_dependency("csv")
```